pkg/policy/chain_test.go

```diff
@@ -17,64 +17,60 @@ limitations under the License.
 package policy
 
 import (
-	"crypto"
 	"crypto/x509"
-	"crypto/x509/pkix"
-	"encoding/pem"
 	"net"
-	"net/url"
 	"testing"
 	"time"
 
 	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
 	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
-	cmpki "github.com/jetstack/cert-manager/pkg/util/pki"
-	apiequality "k8s.io/apimachinery/pkg/api/equality"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
 	cmpolicy "github.com/cert-manager/policy-approver/pkg/api/v1alpha1"
+
+	test "github.com/cert-manager/policy-approver/test/gen"
```

Comment:
We don't need to rename the `gen` package here, and causes conflicts later in.

```suggestion
	"github.com/cert-manager/policy-approver/test/gen"
```

Reply:
Fixed in c1b96d5